Prace_domowe/Praca_domowa3/Grupa1/Gryszkiewicz_Maciej/PD3_Maciej_Gryszkiewicz.ipynb

```diff
@@ -0,0 +1,864 @@
+{
```

Comment:
<p>Accuracy nie jest dobrą miarą gdy klasy są niezbalansowane - bardzo dobrze, że później to zauważono i skomentowano :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>